src/test/java/redis/clients/jedis/tests/commands/ClientIdCommandsTest.java

```diff
@@ -0,0 +1,73 @@
+package redis.clients.jedis.tests.commands;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import redis.clients.jedis.Jedis;
+
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Tests related to clientId
+ */
+public class ClientIdCommandsTest extends JedisCommandTestBase {
```

Comment:
No need to create new class. `ClientCommandsTest` should be enough. So move the test methods in that class.

Reply:
Added test case to existing **ClientCommandsTest**